content/shared/README.md

```diff
@@ -21,5 +20,8 @@ frontmatter.
 5. In the doc body, remove the shared Markdown text and add a comment that points to the shared file, in case someone happens upon the page in the repo--for example, in `/content/3/core/reference/influxql/regular-expressions.md`, add the following:
```

Comment:
```suggestion
5. In the doc body, remove the shared Markdown text and add a comment that points to the shared file, in case someone happens upon the page in the repo--for example, in `/content/influxdb3/core/reference/influxql/regular-expressions.md`, add the following:
```